Externals_CLM.cfg

```diff
@@ -1,8 +1,8 @@
 [fates]
 local_path = src/fates
 protocol = git
-repo_url = https://github.com/NGEET/fates
-tag = sci.1.21.0_api.7.0.0
+repo_url = https://github.com/NCAR/fates-release
```

Comment:
looks good, but was it part of your plant to revert to the NCAR repository also?

Reply:
Whoops. I've moved it back to NGEET.